fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -824,7 +1076,6 @@ private void explicitMergeIndex(Index index, RecordLayerPropertyStorage contextP
                     .build()) {
                 indexBuilder.mergeIndex();
             }
-            commit(context);
```

Comment:
The commit is unnecessary, because the indexer doesn't actually use the index.
In theory this could create a store builder that doesn't have the context at all.
I wanted to remove it to ensure that nothing was using this context.